### PR TITLE
webgui: simply ignore Show() in batch, update most of tutorials

### PR DESCRIPTION
**Pull request by linev**

### CODE REVIEW

File: tutorials/v7/draw_v6.cxx [hunk 1]
```diff
@@ -28,43 +28,36 @@ R__LOAD_LIBRARY(libGpad);
...
-      canvas->SaveAs("draw.png", true,
-                     [](bool res) { std::cout << "Producing PNG done res = " << (res ? "true" : "false") << std::endl; });
-
-      // this function executed in synchronous mode (async = false is default),
-      // mean previous file saving will be completed as well at this point
-      canvas->SaveAs("draw.svg"); // synchronous
-   }
+   // request to create PNG file in asynchronous mode and specify lambda function as callback
+   // when request processed by the client, callback invoked with result value
+   // canvas->SaveAs("draw.png", true,
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_v6.cxx]: I think it's nice to show the async saving. Does this not work in batch mode, or why did you need to comment it?

File: tutorials/v7/draw_v6.cxx [hunk 2]
```diff
@@ -28,43 +28,36 @@ R__LOAD_LIBRARY(libGpad);
...
    // canvas->Show("/usr/bin/chromium --app=$url &"); // one could use $url parameter, which replaced with canvas URL
 
+   canvas->Show(); // new window should popup and async update will be triggered
+
    // synchronous, wait until painting is finished
    canvas->Update(false,
                   [](bool res) { std::cout << "First Update done = " << (res ? "true" : "false") << std::endl; });
 
-   // canvas->Modified(); // when uncommented, invalidate canvas and force repainting with next Update()
+   canvas->Modified(); // when uncommented, invalidate canvas and force repainting with next Update()
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_v6.cxx]: This is uncommented - maybe update the comment?

File: tutorials/v7/fitpanel.cxx [hunk 3]
```diff
@@ -15,7 +15,7 @@
  * For the list of contributors see $ROOTSYS/README/CREDITS.             *
  *************************************************************************/
 
-R__LOAD_LIBRARY(libGpad);
+// R__LOAD_LIBRARY(libGpad);
```
Comment (Axel-Naumann) [inline tutorials/v7/fitpanel.cxx]: If we don't need it then please remove it.

File: tutorials/v7/draw_mt.cxx [hunk 4]
```diff
@@ -57,12 +57,13 @@ void draw_canvas(const std::string &title, RColor col)
    canvas->Draw(pHist)->SetLineColor(col);
    canvas->Draw(pHist2)->SetLineColor(RColor::kBlue);
 
-   bool batch = gROOT->IsWebDisplayBatch();
+   // bool batch = gROOT->IsWebDisplayBatch();
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_mt.cxx]: Remove?

File: tutorials/v7/draw_v6.cxx (hunk 1 above)
Comment (linev) [reply tutorials/v7/draw_v6.cxx]: SaveAs is working also in batch mode.
But if only firefox browser is installed - it will fail for a moment on batch farm without $DISPLAY.
Therefore I left out image saving

**Comment by linev** [comment]
I update comments in the tutorials and remove unused parts

File: tutorials/v7/draw_v6.cxx [hunk 5]
```diff
@@ -28,43 +28,36 @@ R__LOAD_LIBRARY(libGpad);
...
-      canvas->SaveAs("draw.png", true,
-                     [](bool res) { std::cout << "Producing PNG done res = " << (res ? "true" : "false") << std::endl; });
-
-      // this function executed in synchronous mode (async = false is default),
-      // mean previous file saving will be completed as well at this point
-      canvas->SaveAs("draw.svg"); // synchronous
-   }
+   // request to create PNG file in asynchronous mode and specify lambda function as callback
+   // when request processed by the client, callback invoked with result value
+   // canvas->SaveAs("draw.png", true,
```
Comment (Axel-Naumann) [inline tutorials/v7/draw_v6.cxx]: But then why not leave this protected by `if (!gROOT->IsWebDisplayBatch()) {`?

Comment (linev) [reply tutorials/v7/draw_v6.cxx]: > But then why not leave this protected by if (!gROOT->IsWebDisplayBatch()) {

While you do not like tutorials with such statements. 

Let me try to fix problem with generation of firefox profile in batch and than we can add SaveAs() statements in several of our tutorials
